modules/app-applications/src/main/resources/assets/js/app/resource/AdminToolDescriptor.ts

```diff
@@ -0,0 +1,115 @@
+import DescriptorKey = api.content.page.DescriptorKey;
+import AdminToolDescriptorJson = api.content.json.AdminToolDescriptorJson;
+
+export class AdminToolDescriptor {
+
+    private static BASE_TOOL_URI: string = "/admin/tool";
```

Comment:
We should not hardcode this in the code (admin is not necessarily under "/admin/"). I suggest we generate urls on the server (using the same code as ``lib-admin.getToolUrl()``) and return them in the json.